app/models/cemetery.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'common/models/base'
+
+# Cemetery model
+class Cemetery < Common::Base
+  include ActiveModel::Validations
+
+  validates :cemetery_type, inclusion: { in: %w(S N P I A M) }
+  validates :name, :num, presence: true
```

Comment:
Same comment about validation. I want to make sure we're only validating cemetery based on user input, not based on what is provided to us from EOAS